src/pages/LoginPage.jsx

```diff
@@ -0,0 +1,49 @@
+import { useState } from 'react';
+import styles from '../App.module.css';
+import Input from '../components/common/Input';
+import { useDispatch } from 'react-redux';
+import { login as loginAction } from '../store/user/actions';
+import Logo from '../components/Navigation/Logo';
+
+const USER_CREDENTIALS = {
```

Comment:
please consider storing constants in a separate file

Reply:
Done

Reply:
please create index.js in constants folder and export everything from there

Reply:
Done